test/preprocess-bench.py

```diff
@@ -38,11 +45,13 @@
     train_iter = iter(train_loader)
 
     start_time = timer()
-    batch_count = 100 * args.nThreads
-    for i in tqdm(xrange(batch_count)):
+    batch_count = 20 * args.nThreads
+    for _ in tqdm(range(batch_count)):
```

Comment:
If you're going to use range then create it outside of the benchmark loop (it's going to make a full list in Python2).